lib/EnsEMBL/REST/Model/ga4gh/Beacon.pm

```diff
@@ -362,22 +354,29 @@ sub beacon_query {
     $input_coord{'ref_allele'} = $data->{referenceBases};
     $input_coord{'alt_allele'} = $data->{alternateBases} ? $data->{alternateBases} : $data->{variantType};
```

Comment:
What if the `variantType` and `alternateBases` are not given in a [bracket query](http://docs.genomebeacons.org/variant-queries/#beacon-bracket-queries)? See also comment below on defining `$range_query` based on these values.

Reply:
By definition, if none is given then the query matches any type of variants in the coordinates range.